.travis.yml

```diff
@@ -3,8 +3,9 @@ before_install:
   - gem install bundler
 rvm:
   - ruby-head
-  - 2.3.3
-  - 2.2.6
+  - 2.4.0
```

Comment:
Why not 2.4 and let it automatically use the latest 2.4.x instead of locking to 2.4.0?